### PR TITLE
feat: new `installGlobalAutoImports` util

### PR DESCRIPTION
**Pull request by antfu**

### CODE REVIEW

File: src/global.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  await Promise.all(
+    imports.map(async (i) => {
+      if (i.disabled) {
+        return
+      }
+      const as = i.as || i.name
+      if (overrides || !(as in globalObject)) {
+        const module = await import(i.from)
+        globalObject[as] = module[i.name]
```
Comment (pi0) [inline src/global.ts:25]: (do we need to add a sanity check and avoid assignment if `i.name` not exists in module?

File: src/types.ts [hunk 2]
```diff
@@ -156,3 +156,16 @@ export interface Addon {
...
+  /**
+   * @default globalThis
+   */
+  globalObject?: any
+
+  /**
+   * Overrides the existing property
+   * @default false
+   */
+  overrides?: boolean
```
Comment (pi0) [inline src/types.ts:170]:
```suggestion
  overrideGlobal?: boolean
```

File: src/global.ts (hunk 1 above)
Comment (antfu) [reply src/global.ts:25]: I think that makes sense. But if we do so, should we warn that the name does not exists, or ignore silently?

File: src/types.ts (hunk 2 above)
Comment (antfu) [reply src/types.ts:170]: I am not sure if I got the semantic, you are trying to improve, can you explain a bit more? I feel the `global` is already presented in the function name and changing this creates a little duplication to me personally.

**Comment by antfu** [comment]
Merge it for now, and leave for future improvements

**Pull request merged**